assets/js/featured-resource-preview.js

```diff
@@ -3,6 +3,18 @@
    ========================================================================== */
 
 (function($) {
+  function convertToNbsp(text) {
+    // Any change here should also be made in zds/utils/templatetags/french_typography.py
+    return text
+      .replace(' ;', '&#8239;;')
+      .replace(' ?', '&#8239;?')
+      .replace(' !', '&#8239;!')
+      .replace(' %', '&#8239;%')
+      .replace('« ', '«&nbsp;')
+      .replace(' »', '&nbsp;»')
+      .replace(' :', '&nbsp;:');
```

Comment:
Pas de point-virgule.
```suggestion
      .replace(' :', '&nbsp;:')
```

Reply:
Fait. Mais ça m'étonne que le point-virgule soit carrément interdit...